packages/server/graphql/mutations/updatePromptResponse.ts

```diff
@@ -0,0 +1,78 @@
+import {JSONContent} from '@tiptap/core'
+import {GraphQLID, GraphQLNonNull, GraphQLString} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {getTeamPromptResponseById} from '../../postgres/queries/getTeamPromptResponsesByIds'
+import {updateTeamPromptResponseContentById} from '../../postgres/queries/updateTeamPromptResponseContentById'
+import {getUserId} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import {GQLContext} from '../graphql'
+import UpdatePromptResponsePayload from '../types/UpdatePromptResponsePayload'
+
+const extractTextFromTipTapJSONContent = (input: JSONContent) => {
+  if (input.text) {
+    return input.text
+  } else if (input.content) {
+    const content = input.content
+    const newLine = input.type === 'paragraph' ? '\n' : ''
+    return (
+      content.map((subContent) => extractTextFromTipTapJSONContent(subContent)).join('') + newLine
+    )
+  } else {
+    return input.type === 'paragraph' ? '\n' : ''
+  }
+}
```

Comment:
+1 I'd create a separate directory in `utils` named `tiptap` and would put it there